src/plugins/plugin.legend.js

```diff
@@ -622,7 +622,7 @@ export default {
 
     labels: {
       color: (ctx) => ctx.chart.options.color,
-      boxWidth: 40,
+      // boxWidth: default is 40,
```

Comment:
This would become a breaking change for users

Reply:
default value has been transferred to getBoxSize function.
`let {boxHeight = fontSize, boxWidth = 40} = labelOpts;`

Reply:
@touletan that is a breaking change, because that default is no longer configurable.